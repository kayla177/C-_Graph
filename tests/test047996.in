ENTITY KR2A1OD4 name_KR2A journal
ENTITY RGPXFACQ name_RGPX institution
ENTITY 1L81ZDQC name_1L81 author
RELATIONSHIP KR2A1OD4 presented_at RGPXFACQ 9.5
PRINT KR2A1OD4
HIGHEST
EXIT
