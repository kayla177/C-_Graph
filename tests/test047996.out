success
success
success
success
RGPXFACQ
KR2A1OD4 RGPXFACQ 9.5
